x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -169,9 +187,7 @@ export function ServiceInventory() {
     ...preloadedServices,
   ].some((item) => 'healthStatus' in item);
 
-  const tiebreakerField = useKibana().services.uiSettings?.get<boolean>(
-    apmServiceInventoryOptimizedSorting
-  )
+  const tiebreakerField = useOptimizedSorting
```

Comment:
Maybe, but I like having `tiebreakerField` to make it more explicit what the effect actually is on sorting, rather than passing a very high-level and non-descriptive variable name down to very scoped functions who probably shouldn't know anything about "optimized sorting".